rapidtide/util.py

```diff
@@ -28,13 +29,16 @@
 import matplotlib.pyplot as plt
 import numpy as np
 import pyfftw
+import pyfftw.interfaces.scipy_fftpack as fftpack
 from numba import jit
-from scipy import fftpack
 
 import rapidtide._version as tide_versioneer
 import rapidtide.io as tide_io
 
-fftpack = pyfftw.interfaces.scipy_fftpack
+LGR = logging.getLogger(__name__)
+TimingLGR = logging.getLogger("TIMING")
+MemoryLGR = logging.getLogger("MEMORY")
```

Comment:
These three will need to go in each file, along with `import logging`.

Reply:
Ok, so we try them out here, then assuming nothing breaks, we put it everywhere?

Reply:
Pretty much! It's the same structure we use in `tedana`, only instead of a general logger, a timing logger, and a memory logger, we have a general logger, a report logger (for building a citable description of the steps in the workflow), and a references logger (in which references are added in step with the report logger). If this works out then it should be straightforward to create report and reference loggers to handle #32 as well.